### PR TITLE
Feature/foward

### PR DESCRIPTION
**Pull request by DGiulietti**
Added a forward to our story.

### CODE REVIEW

File: forward.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Forward
+## written by Diana Giulietti
+
+This is a story written in collaboration between Derek and Diana. It has taken a total of 2 minutes to write.
```
Comment (derekprovance) [inline forward.md:4]: I actually think this should be Diana and Derek

**Review by derekprovance — CHANGES_REQUESTED** [review]
BLAABHAHBABALBAH

File: forward.md (hunk 1 above)
Comment (DGiulietti) [reply forward.md:4]: THEN PAY ME MORE MONIES!

Comment (derekprovance) [reply forward.md:4]: NOPE, FIX IT

**Comment by derekprovance** [comment]
Yep Reviewed it and everything looks good!

**Pull request merged**